src/Network/Algorand/Node/Api/Content.hs

```diff
@@ -0,0 +1,57 @@
+-- SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+-- | The REST API v2 of Algod, the Algorand node.
+--
+-- See <https://developer.algorand.org/docs/reference/rest-apis/algod/v2/>
```

Comment:
This description was copied from `Api.hs` and should be updated to something more specific. Same for `Type.hs`.